core/trino-main/src/main/java/io/trino/operator/aggregation/LongSumAggregation.java

```diff
@@ -38,10 +38,11 @@ public static void sum(@AggregationState LongLongState state, @SqlType(StandardT
     }
 
     @RemoveInputFunction
-    public static void removeInput(@AggregationState LongLongState state, @SqlType(StandardTypes.BIGINT) long value)
+    public static boolean removeInput(@AggregationState LongLongState state, @SqlType(StandardTypes.BIGINT) long value)
     {
         state.setFirst(state.getFirst() - 1);
         state.setSecond(BigintOperators.subtract(state.getSecond(), value));
+        return true;
```

Comment:
Add an inline comment about how state should never be infinite / NaN for long inputs

Reply:
Done